spec/services/billable_metrics/aggregations/sum_service_spec.rb

```diff
@@ -212,11 +216,42 @@
     end
 
     it 'aggregates the events' do
-      result = sum_service.aggregate(from_datetime: from_datetime, to_datetime: to_datetime, options: options)
+      result = sum_service.aggregate(from_datetime:, to_datetime:, options:)
 
       expect(result.aggregation).to eq(20)
       expect(result.count).to eq(2)
       expect(result.options).to eq({ running_total: [12, 20] })
     end
   end
+
+  context 'when event is given' do
+    let(:instant_event) do
+      create(
+        :event,
+        code: billable_metric.code,
+        customer:,
+        subscription:,
+        timestamp: Time.zone.now - 1.day,
+        properties:,
+      )
+    end
+
+    let(:properties) { { total_count: 12 } }
+
+    it 'assigns an instant aggregation' do
+      result = sum_service.aggregate(from_datetime:, to_datetime:)
+
+      expect(result.instant_aggregation).to eq(12)
+    end
+
+    context 'when event is missing properties' do
```

Comment:
maybe another exemple with a given properties not matching the expected field_name?